src/components/icon/icon.stories.mdx

```diff
@@ -73,6 +73,8 @@ such as `tooltipPosition`, `tooltipVisible` , `tooltipBgColor`, `tooltipFontColo
 
 ### Various bgSizes
 
+If using the `bgSize` prop in combination with `fontSize`, the `fontSize` prop value will take precedence and the background size may be adjusted to accommodate for a larger icon. 
```

Comment:
```suggestion
If using the `bgSize` prop in combination with `fontSize`, the `fontSize` prop value will take precedence and the background size may be adjusted to accommodate a larger icon. 
```

**suggestion** (non-blocking): may just be me but "accomodate for" seems weird, I think it reads fine without the "for"

Reply:
I agree, adding a new story to demonstrate this might be the better option.